### ISSUE TITLE
[Mechanic] Disable or Increase Maximum Hopper per Chunk Limit

### ISSUE THREAD

**Issue by Basically-Andrew** [EarthMC/Issue-Tracker#1500]
### Is your feature request related to a problem? Please describe.

It is practically impossible and unpractical to create a machine on EarthMC Aurora with a limit of 64x hoppers per chunk.

### Describe the solution you'd like

Either remove this mechanic entirely or raise the hopper limit to a number like 128.

### Additional context

Simple machines like an alchemy grinder or chest sorter won't work in a single chunk.

**Comment by Nexus9594** [comment]
It's to reduce server lag, the server has enough stress already that rasing the hopper count from "64" to "128" could cause for possible lag and or lag machines. For example, let's say you make a cheat sorter that would require that amount of hoppers and items coming in and out it would slow everything down. Since hoppers count (somewhat) as an entity (code wise) it causes more lag that just a normal block. Yes, assuming earth mc has some powerful servers theoretically it would handle many a couple chunks of that, once people start getting used to the idea of a 128 hopper limit or no limit things start to go crazy, it would cause server wide lag. There would also be the Minecraft player who decides they want to crash/lag the server. I personally think 64 hoppers is a good limit per chunk, unless the earth MC servers are powerful enough to support this type of stuff I think it should stay at 64. In the end up to the staff, that's just my take.

**Comment by Basically-Andrew** [comment]
I agree with everything you said, but it's ridiculous any simple Redstone machine won't work in a single chunk. Mcmmo requires over 64 Hoppers to grind Alchemy, so this really can discourage players.

**Comment by Warriorrrr** [comment]
The limit will remain at 64 for the time being, since hoppers already cause a lot of lag. Other than nerfing hopper speed significantly, limiting the amount per chunk is the only way to prevent people from building large machines with hoppers.
